samples/subsys/fs/littlefs/boards/frdm_mcxa156.conf

```diff
@@ -0,0 +1,5 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_FS_LITTLEFS_PROG_SIZE=128
+CONFIG_FS_LITTLEFS_CACHE_SIZE=128
```

Comment:
Just to make sure, because documentation and Kconfig description fails to mention that: you do know that these are defaults for instances configured with macro `FS_LITTLEFS_DECLARE_DEFAULT_CONFIG` and DTS instances that did not provide some of params, right?